scripts/ssl_tts/ssl_tts_vc.py

```diff
@@ -0,0 +1,254 @@
+import argparse
+import os
+
+import librosa
+import soundfile
+import torch
+
+from nemo.collections.asr.parts.preprocessing.features import WaveformFeaturizer
+from nemo.collections.tts.models import fastpitch_ssl, hifigan, ssl_tts
+from nemo.collections.tts.torch.helpers import get_base_dir
+
+
+def load_wav(wav_path, wav_featurizer, pad_multiple=1024):
+    wav = wav_featurizer.process(wav_path)
+    if wav.shape[0 % pad_multiple] != 0:
```

Comment:
Should the modulo be outside of the brackets?

Reply:
fixed.